pkg/systemd/services/services.jsx

```diff
@@ -503,11 +507,11 @@
                                             this.processFailedUnits();
                                         });
                             }, ex => {
-                                this.props.isLoading(false);
+                                this.props.SetIsLoading(false);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18411-20230227-080740-f45e3de4-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)